tests/src/test/java/alluxio/client/cli/fs/FileSystemShellUtilsTest.java

```diff
@@ -125,9 +126,11 @@ public static String resetFileHierarchy(FileSystem fs, WritePType writeType)
     if (fs.exists(new AlluxioURI(TEST_DIR))) {
       fs.delete(new AlluxioURI(TEST_DIR), DeletePOptions.newBuilder().setRecursive(true).build());
     }
-    fs.createDirectory(new AlluxioURI(TEST_DIR));
-    fs.createDirectory(new AlluxioURI(TEST_DIR + "/foo"));
-    fs.createDirectory(new AlluxioURI(TEST_DIR + "/bar"));
+    CreateDirectoryPOptions dirOptions = CreateDirectoryPOptions.getDefaultInstance().toBuilder()
```

Comment:
Why did you have to change this now?

Reply:
Because the client configuration write type is set as `CACHE_THROUGH` and when the directories are created we expect them to be in a not-persisted state. Without passing the write type here the directories weren't being created with the proper write type and the assertion was failing for tests which checked that the directory wasn't persisted after creating the file hierarchy